src/create-transformer.ts

```diff
@@ -47,30 +48,11 @@ export function createTransformer<A, B>(
 
     // Memoizes: object -> reactive view that applies transformer to the object
     const views = new Map<A, IComputedValue<B>>()
-    let onCleanup: Function | undefined = undefined
-    let keepAlive: boolean = false
-    let debugNameGenerator: Function | undefined = undefined
-    if (typeof arg2 === "object") {
-        onCleanup = arg2.onCleanup
-        keepAlive = arg2.keepAlive !== undefined ? arg2.keepAlive : false
-        debugNameGenerator = arg2.debugNameGenerator
-    } else if (typeof arg2 === "function") {
-        onCleanup = arg2
-    }
+    const opts = getOpts(arg2)
+    const { debugNameGenerator, keepAlive, onCleanup } = opts
 
     function createView(sourceObject: A) {
         let latestValue: B
-        let computedValueOptions = {}
-        if (typeof arg2 === "object") {
-            onCleanup = arg2.onCleanup
-            debugNameGenerator = arg2.debugNameGenerator
-            computedValueOptions = arg2
-        } else if (typeof arg2 === "function") {
-            onCleanup = arg2
-        } else {
-            onCleanup = undefined
-            debugNameGenerator = undefined
-        }
```

Comment:
I needed to get `requiresReaction` from the arguments passed to the `createTransformer`. While looking at how to do that I noticed that this built `computedValueOptions` appears to be unnecessary. I thought it was being done to prevent `onCleanup` from passing down to the `computed`, but it assigns `arg2` to `computedValueOptions` on L67 anyways...